editor/app/crashes/[record_locator]/page.tsx

```diff
@@ -34,6 +35,14 @@ export default function CrashDetailsPage({
 }) {
   const recordLocator = params.record_locator;
 
+  // Handles scrolling down to element on key press
+  const onKeyPress = (event: KeyboardEvent) => {
+    const element = document.getElementById(String(event.key));
+    element?.scrollIntoView({ behavior: "smooth" });
```

Comment:
i would typically prefer the `smooth` behavior, but because we're trying to enable super fast QA/QC, i think the default scroll behavior would be best. we can also use Xavier as a test subject...